sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3736,6 +3752,328 @@ SUBROUTINE MISCLN
+                ELSE
+                   GRID1(I,J)=D00
+                ENDIF
+               ENDDO
+            ENDDO
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(990))
+              fld_info(cfld)%lvl=LVLSXML(1,IGET(990))
+! $omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = GRID1(i,jj)
+                enddo
+              enddo
+             endif
+           ENDIF
+
+!Effective Layer Supercell Parameter
+            IF (IGET(991)>0) THEN
+            DO J=JSTA,JEND
+               DO I=1,IM
+                IF (ESHR(I,J)<10.) THEN
+                   ESHRtmp=D00
+                ELSEIF (ESHR(I,J)>20.0) THEN
+                   ESHRtmp=1
+                ELSE
+                   ESHRtmp=(ESHR(I,J)/20.)
+                ENDIF
+                IF (MUCIN(I,J)>-40.) THEN
+                   MUCINtmp=1.0
+                ELSE
+                   MUCINtmp=(-40./MUCIN(I,J))
+                ENDIF
+                STP=(MUCAPE(I,J)/D1000)*(ESRH(I,J)/50.)*&
+                        ESHRtmp*MUCINtmp
+                IF (STP>0) THEN
+                   GRID1(I,J)=STP
+                ELSE
+                   GRID1(I,J)=D00
+                ENDIF
+               ENDDO
+            ENDDO
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(991))
+              fld_info(cfld)%lvl=LVLSXML(1,IGET(991))
+! $omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = GRID1(i,jj)
+                enddo
+              enddo
+             endif
+           ENDIF
+
+!Mixed Layer (100 mb) Virtual LFC
+
+           IF (IGET(992)>0) THEN
+!$omp parallel do private(i,j)
+           DO J=JSTA,JEND
+             DO I=1,IM
+               EGRID1(I,J) = -H99999
+               EGRID2(I,J) = -H99999
+               EGRID3(I,J) = -H99999
+               EGRID4(I,J) = -H99999
+               EGRID5(I,J) = -H99999
+               EGRID6(I,J) = -H99999
+               EGRID7(I,J) = -H99999
+               EGRID8(I,J) = -H99999
+               LB2(I,J)  = (LVLBND(I,J,1) + LVLBND(I,J,2) +           &
+                            LVLBND(I,J,3))/3
+               P1D(I,J)  = (PBND(I,J,1) + PBND(I,J,2) + PBND(I,J,3))/3
+               T1D(I,J)  = (TVIRTUAL(TBND(I,J,1),QBND(I,J,1)) +       &
+                            TVIRTUAL(TBND(I,J,2),QBND(I,J,2)) +       &
+                            TVIRTUAL(TBND(I,J,3),QBND(I,J,3)))/3
+               Q1D(I,J)  = (QBND(I,J,1) + QBND(I,J,2) + QBND(I,J,3))/3
+             ENDDO
+           ENDDO
+
+             DPBND = 0.
+             ITYPE = 2
+! EGRID3 is Virtual LFC
+             CALL CALCAPE2(ITYPE,DPBND,P1D,T1D,Q1D,LB2,            &
+                           EGRID1,EGRID2,EGRID3,EGRID4,EGRID5,     &
```

Comment:
Hi Ed,

Is this Q1D the mixing ratio of the Most Unstable parcel? calculated when ITYPE=2 only?

Reply:
Hi Ed,
For using Q1D here as the Most Unstable mixing ratio, I am not sure about this. 
But to help the merging be done soon, I just comment and have the code pass the review.
We may take a look at it later.
thanks!
- Gang